package.json

```diff
@@ -64,7 +64,6 @@
     "@typescript-eslint/parser": "4.3.0",
     "babel-eslint": "10.1.0",
     "common-tags": "1.8.0",
-    "eslint": "7.10.0",
```

Comment:
eslint had to be removed from devDependencies because it was causing the following yarn warning:

```
warning package.json: "dependencies" has dependency "eslint" with range "^7.10.0" that collides with a dependency in "devDependencies" of the same name with version "7.10.0"
```

Reply:
:+1: the widened version exists as a dependency, so it still works! :+1: thanks!